src/dropin-model.js

```diff
@@ -205,13 +205,20 @@ function getSupportedPaymentOptions(options) {
 
 function isPaymentOptionEnabled(paymentOption, options) {
   var gatewayConfiguration = options.client.getConfiguration().gatewayConfiguration;
+  var applePayEnabled, applePayBrowserSupported;
 
   if (paymentOption === 'card') {
     return gatewayConfiguration.creditCards.supportedCardTypes.length > 0;
   } else if (paymentOption === 'paypal') {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypal);
   } else if (paymentOption === 'paypalCredit') {
     return gatewayConfiguration.paypalEnabled && Boolean(options.merchantConfiguration.paypalCredit);
+  } else if (paymentOption === 'applePay') {
+    applePayEnabled = gatewayConfiguration.applePay && Boolean(options.merchantConfiguration.applePay);
+    applePayBrowserSupported = window.ApplePaySession && ApplePaySession.canMakePayments(); // eslint-disable-line no-undef
```

Comment:
We should use `global` instead of `window`. And if we use `global` for the `canMakePayment` function, we won't have to use `eslint-disable-line`